src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -342,12 +351,13 @@ class OrganizationIntegrations extends AsyncComponent<
     const {organization} = this.props;
 
     return (
-      <IntegrationDirectorySentryAppRow
+      <IntegrationRow
```

Comment:
@NisanthanNanthakumar Now that we are using a single component, do we really need to instantiate it in three different places? I would think we could just use it once and simplify our code.

Reply:
@scefali the logic that generates the props are very different. I think it's more readable when the logic is separated out.